inspirehep/modules/theme/views.py

```diff
@@ -101,3 +116,40 @@ def data():
 @blueprint.route('/ping')
 def ping():
     return 'OK'
+
+
+#
+# Feedback
+#
+
+@blueprint.route('/postfeedback', methods=['POST', ])
+def postfeedback():
+    """Handler to create a ticket from user feedback."""
+    feedback = request.form.get('feedback')
+    if feedback == '':
+        return jsonify(success=False), 400
+
+    replytoaddr = request.form.get('replytoaddr', None)
+    if replytoaddr is None:
+        if current_user.is_anonymous:
+            return jsonify(success=False), 403
+        else:
+            replytoaddr = current_user.get('email')
+            if replytoaddr == '':
+                return jsonify(success=False), 403
+
+    content = 'Feedback:\n{feedback}'.format(feedback=feedback)
+    message = {
+        'sender': current_app.config['CFG_SITE_SUPPORT_EMAIL'],
+        'recipients': [current_app.config['INSPIRELABS_FEEDBACK_EMAIL']],
+        'subject': 'INSPIRE Labs Feedback',
+        'body': content,
+        'reply_to': replytoaddr
+    }
+
+    sending = send_email.delay(message)
+
+    if sending.failed():
```

Comment:
That looks nice. I guess it is still async so not sure what failed would mean (worker not available?), We might say to the user that it's all good and then the task fail while sending the email (like wrong email address or something). But that's the point of async, isn't it?